CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
+- Added check for invalid zim file names
```

Comment:
ZIM is an acronym, it must be spelled all uppercase in text (not in variables which must be lowercase due to Python conventions)

Reply:
Fixed in [00823a9](https://github.com/openzim/warc2zim/pull/234/commits/00823a9a4c0cfe42bb3b75c6d7ed450519933439)